site/content/resources/_index.html

```diff
@@ -30,6 +30,24 @@ <h3>Here you will find external resources about Carvel, such as videos, workshop
            <p><a href="https://tanzu.vmware.com/developer/workshops/lab-getting-started-with-carvel/">Getting Started with Carvel (formerly k14s)</a>.</p>
        </div>
    </div>
+   <div class="col">
+       <div class="icon">
+           <img src="/img/logo.svg" />
+       </div>
+       <div class="content">
+           <p class="strong">VMware Tanzu Kubernetes Grid 1.4 Announcement Blog</p>
+           <p><a href="https://tanzu.vmware.com/content/blog/general-availability-vmware-tanzu-kubernetes-grid-1-4">VMware Tanzu Kubernetes Grid 1.4 migrates all extensions and add-ons using Carvel Package Management APIs</a>.</p>
```

Comment:
i dont think this counts as a resource since it doesnt actually talk about details (just says packages are avilable).

Reply:
Yeah, I agree. I added it here for now till we figure out how to showcase Carvel adoption. That might probably be a page on its own. 

I can remove this one for now.

Reply:
Removed!